jvm/src/main/scala/fm/common/EmailSender.scala

```diff
@@ -100,4 +224,27 @@ final case class EmailSender(user: String, pass: String, host: String) {
       transport.close()
     }
   }
+
+  private def createMultipart(plaintextBody: String, htmlBody: String): MimeMultipart = {
+    val multipart: MimeMultipart = new MimeMultipart()
+
+    // According to the RFC for Multipart content, the plainest format should be set first and the richest format last
```

Comment:
Typo: `plainest ` should be `plaintext `

Reply:
Oh I just copied that part from w3. They ranked it from plainiest to richest.